### ISSUE TITLE
v3.5.2 npm broken (probably due to npm-publish.yml)

### ISSUE THREAD

**Issue by icflorescu** [atulmy/gql-query-builder#54]
Hey @atulmy,

I'm not sure why or how, but something is really broken in the `v3.5.2` package published on npm.
Just try `npm i gql-query-builder` or `yarn add gql-query-builder` in an empty project and you'll see the only files you end up with in `node_modules/gql-query-builder` are these:

```
-rw-r--r-- 1 user user 1.1K Dec  5 12:49 LICENSE
-rw-r--r-- 1 user user  13K Dec  5 12:49 README.md
-rw-r--r-- 1 user user 1.5K Dec  5 12:49 package.json
```

There must be something wrong with your [npm-publish.yml](https://github.com/atulmy/gql-query-builder/blob/master/.github/workflows/npm-publish.yml) workflow.
Let me know if there's anything I can do to help.

**Comment by atulmy** [comment]
Thanks for reporting and sorry about the issue, took a while to wrap my head around GitHub Actions CI/CD setup. 

Resolved, v3.5.4 should work.

**Comment by icflorescu** [comment]
Thanks for your quick response!
